stan/math/prim/err/check_simplex.hpp

```diff
@@ -21,12 +24,7 @@ namespace math {
  * specified by <code>CONSTRAINT_TOLERANCE</code>. This function
  * only accepts Eigen vectors, statically typed vectors, not
  * general matrices with 1 column.
- * @tparam T Scalar type of the vector
- * @tparam R Eigen row type, either 1 if we have a row vector
- *         or -1 if we have a column vector.
- * @tparam C Eigen column type, either 1 if we have a column vector
- *         or -1 if we have a row vector. Moreover, we either have
- *         R = 1 and C = -1 or R = -1 and C = 1.
+ * @tparam T A type inheriting from EigenBase.
```

Comment:
[optional]
Only sentences should have periods after them.  Sorry the doc's already so inconsistent.